lib/action_account.rb

```diff
@@ -63,11 +69,67 @@ def log_text
         error_activating: "There was an error activating the user's profile. Please try again.",
         past_eligibility: 'User is past the 30 day review eligibility.',
         missing_uuid: 'Error: Could not find user with that UUID',
+        user_suspended: 'User has been suspended',
+        user_reinstated: 'User has been reinstated',
+        user_already_suspended: 'User has already been suspended',
+        user_is_not_suspended: 'User is not suspended',
       }
     end
   end
 
-  class ReviewReject < ReviewAction
+  module UserActions
+    include LogBase
+
+    def perform_user_action(args:, config:, action:)
+      table = []
+      messages = []
+      uuids = args
+      table << %w[uuid status]
+
+      users = User.where(uuid: uuids).order(:uuid)
+      users.each do |user|
+        log_texts = []
+        if action == :suspend && user.suspended?
+          log_texts << log_text[:user_already_suspended]
+        elsif action == :reinstate && !user.suspended?
+          log_texts << log_text[:user_is_not_suspended]
+        else
+          user.public_send("#{action}!")
+          log_texts << (action == :suspend ? log_text[:user_suspended] : log_text[:user_reinstated])
```

Comment:
I'd consider something like a `case ... when` here because it can make it clearer that if we ever add more actions, which spots in the codebase we need to upgrade (it can be easy to miss a ternary like this)